shopinvader/tests/common.py

```diff
@@ -146,6 +146,9 @@ def setUpClass(cls):
             context=dict(cls.env.context, tracking_disable=cls.tracking_disable)
         )
         CommonMixin._setup_backend(cls)
+        # It seem that setUpComponent / setUpRegistry loose stuff from
```

Comment:
how?

Reply:
no idea ! I just tested it.
You can try to read a computed field (like the url_key) before the setup and after.
Before you have the value after it's not here anymore (crazy thing).

Reply:
If I flush before then all data are keep.
I will try to investigate later, but if you have the time, help are welcome

Reply:
but then wouldn't be better to call `self.env["base"].flush()` and add TODO for later?

Reply:
yes better ! Done